src/main/java/com/blogify/blogapi/model/Category.java

```diff
@@ -29,6 +30,6 @@ public class Category implements Serializable {
   @CreationTimestamp
   @Getter(AccessLevel.NONE)
   private Instant creationDatetime;
-  @ManyToMany(mappedBy = "categories")
-  private List<User> users;
+  @OneToMany(mappedBy = "category")
```

Comment:
Add `cascade=CascadeType.ALL` in order to persist UserCategory when saving Category

Reply:
okay